specification/recoveryservices/resource-manager/Microsoft.RecoveryServices/stable/2021-01-01/examples/CheckNameAvailability_Available.json

```diff
@@ -0,0 +1,19 @@
+{
+  "parameters": {
+    "subscriptionId": "77777777-b0c6-47a2-b37c-d8e65a629c18",
+    "resourceGroupName": "resGroupFoo",
+    "api-version": "2021-01-01",
```

Comment:
While adding new version the recommendation is to copy the existing version into new directory structure for first commit and then push new changes for easy review. Please create new branch with this recommendation for faster review.

Reply:
@RamyasreeChakka , apart from adding the files for new API-version, I've only modified readme.md files to include newer API version and change the default version in the readme files to the latest version.

Reply:
My apologies @RamyasreeChakka for selecting the tickbox - 'update existing version to fix swagger quality issue in s360'

I dont need to select this checkbox and I've unselected it.

Can you please review this PR?